lib/ConstExtract/ConstExtract.cpp

```diff
@@ -119,28 +120,68 @@ parseProtocolListFromFile(StringRef protocolListFilePath,
   return true;
 }
 
+static std::string extractLiteralOutput(Expr *expr) {
+  std::string LiteralOutput;
+  llvm::raw_string_ostream OutputStream(LiteralOutput);
+  expr->printConstExprValue(&OutputStream, nullptr);
+
+  return LiteralOutput;
+}
+
 static std::shared_ptr<CompileTimeValue>
 extractPropertyInitializationValue(VarDecl *propertyDecl) {
   auto binding = propertyDecl->getParentPatternBinding();
   if (binding) {
     auto originalInit = binding->getOriginalInit(0);
     if (originalInit) {
-      std::string LiteralOutput;
-      llvm::raw_string_ostream OutputStream(LiteralOutput);
-      originalInit->printConstExprValue(&OutputStream, nullptr);
-      if (!LiteralOutput.empty())
+      auto LiteralOutput = extractLiteralOutput(originalInit);
```

Comment:
Interesting, I hadn't noticed that `Expr::printConstExprValue()` returned `false` for some types. It looks like `ExprKind::Call` is one of those, so will this get skipped over by `extractLiteralOutput()` returning `nullptr`?

Reply:
In that case, I believe `extractLiteralOutput` will return an empty `std:string`. With unhanded expressions `Expr::printConstExprValue()` will skip writing any additional information into the output stream + string.